### PR TITLE
Feature/gameplaytask character landed

### PR DESCRIPTION
**Pull request by VectorGamez**

### CODE REVIEW

File: Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#include "Tasks/GASExtAT_WaitCharacterLanded.h"
+
+UGASExtAT_WaitCharacterLanded * UGASExtAT_WaitCharacterLanded::WaitCharacterLanded( UGameplayAbility * owning_ability, bool b_end_on_landed /*= true*/ )
```
Comment (TheEmidee) [inline Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp:3]: end_on_landed (no b_ prefix)

File: Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    return my_obj;
+}
+
+void UGASExtAT_WaitCharacterLanded::Activate()
+{
+    Super::Activate();
+
+    SetWaitingOnAvatar();
+
+    ACharacter * character = Cast< ACharacter >( GetAvatarActor() );
```
Comment (TheEmidee) [inline Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp:16]: combine the 2 lines

File: Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    if ( bEndOnLanded )
+    {
+        EndTask();
+    }
+}
+
+void UGASExtAT_WaitCharacterLanded::OnDestroy( const bool ability_ended )
+{
+    ACharacter * character = Cast< ACharacter >( GetAvatarActor() );
```
Comment (TheEmidee) [inline Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp:39]: combine the 2 lines

**Review by TheEmidee — CHANGES_REQUESTED** [review]

File: Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp (hunk 3 above)
Comment (VectorGamez) [reply Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp]: Done.

File: Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp (hunk 2 above)
Comment (VectorGamez) [reply Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp]: Done.

File: Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp (hunk 1 above)
Comment (VectorGamez) [reply Source/GASExtensions/Private/Tasks/GASExtAT_WaitCharacterLanded.cpp]: Done.

**Review by TheEmidee — APPROVED** [review]

**Pull request merged**